home/blocks.py

```diff
@@ -13,25 +14,40 @@ def render_basic(self, value, context=None):
         if not value:
             return ''
 
+        video_not_supported_text = _("Your browser does not support the video tag.")
+        audio_not_supported_text = _("Your browser does not support the audio element.")
+        # Translators: Part of this message (between %(start_link)s and %(end_link)s ) is a clickable download link
+        download_video_text = _('If you cannot view the above video, perhaps would'
+                ' you like to %(start_link)s download it? %(end_link)s.') % {
+                        'start_link': '<a href={1} download>',
+                        'end_link': '</a>'
+                }
```

Comment:
I don't know if this would translate well (and don't know the best practice to do this yet). Have you tested this?

Reply:
Yes, this works (as long as we use gettext rather than gettext_lazy, see definition of _ at the top).